### ISSUE TITLE
ASHLAR issue on PCA samples

### ISSUE THREAD

**Issue by DenisSch** [labsyspharm/mcmicro#91]
```
Command error:
  WARNING: Stage coordinates' measurement unit is undefined; assuming μm.
  WARNING: Stage coordinates' measurement unit is undefined; assuming μm.
  WARNING: Stage coordinates' measurement unit is undefined; assuming μm.
  Traceback (most recent call last):
    File "/n/groups/lsp/mcmicro/conda/ashlar/bin/ashlar", line 8, in <module>
      sys.exit(main())
    File "/n/groups/lsp/mcmicro/conda/ashlar/lib/python3.6/site-packages/ashlar/scripts/ashlar.py", line 174, in main
      args.quiet
    File "/n/groups/lsp/mcmicro/conda/ashlar/lib/python3.6/site-packages/ashlar/scripts/ashlar.py", line 237, in process_single
      **mosaic_args_final
    File "/n/groups/lsp/mcmicro/conda/ashlar/lib/python3.6/site-packages/ashlar/reg.py", line 935, in __init__
      self._load_correction_profiles(dfp_path, ffp_path)
    File "/n/groups/lsp/mcmicro/conda/ashlar/lib/python3.6/site-packages/ashlar/reg.py", line 1007, in _load_correction_profiles
      self.dfp = self._load_single_profile(dfp_path, num_channels, img_size, 'dark')
    File "/n/groups/lsp/mcmicro/conda/ashlar/lib/python3.6/site-packages/ashlar/reg.py", line 995, in _load_single_profile
      profile_type.capitalize(), profile.shape, img_size
  ValueError: Dark-field profile shape (1080, 1280, 4) does not match target image shape (1080, 1280).

Work dir:
  /n/scratch2/ds230/raw_Z170/c9/0441e7a4f76911e46174c28bf84c82

Tip: when you have fixed the problem you can continue the execution adding the option `-resume` to the run command line
```

**Comment by ArtemSokolov** [comment]
Can you list the content of `/n/scratch2/ds230/raw_Z170/c9/0441e7a4f76911e46174c28bf84c82/.command.sh`?

**Comment by DenisSch** [comment]
```
#!/bin/bash -ue
ashlar Z170_1@20200309_150950_526258.rcpnl Z170_1@20200311_175124_119188.rcpnl Z170_1@20200313_175126_381593.rcpnl Z170_1@20200316_150528_638167.rcpnl Z170_1@20200317_195758_140037.rcpnl Z170_1@20200318_2
13433_804963.rcpnl Z170_1@20200319_205635_343174.rcpnl Z170_1@20200320_224824_931820.rcpnl -m 30 --pyramid --ffp Z170_1@20200313_175126_381593-ffp.tif Z170_1@20200320_224824_931820-ffp.tif Z170_1@20200319
_205635_343174-ffp.tif Z170_1@20200316_150528_638167-ffp.tif Z170_1@20200318_213433_804963-ffp.tif Z170_1@20200317_195758_140037-ffp.tif Z170_1@20200311_175124_119188-ffp.tif Z170_1@20200309_150950_526258
-ffp.tif --dfp Z170_1@20200313_175126_381593-dfp.tif Z170_1@20200320_224824_931820-dfp.tif Z170_1@20200319_205635_343174-dfp.tif Z170_1@20200316_150528_638167-dfp.tif Z170_1@20200318_213433_804963-dfp.tif
 Z170_1@20200317_195758_140037-dfp.tif Z170_1@20200311_175124_119188-dfp.tif Z170_1@20200309_150950_526258-dfp.tif -f Z170_1.ome.tif```

**Comment by DenisSch** [comment]
@jmuhlich Same error on all PCA samples with ```--ilum```

**Comment by jmuhlich** [comment]
Believed to be fixed by #92.
